modules/custom/social_graphql/src/GraphQL/EntityConnection.php

```diff
@@ -247,6 +250,7 @@ protected function getResult() : SyncPromise {
    *   A promise that resolves to the edges of this connection.
    */
   protected function execute() : SyncPromise {
+    /** @var \Drupal\Core\Entity\Query\QueryBase $query */
     $query = $this->queryHelper->getQuery();
```

Comment:
[Prefer assertions](https://phpstan.org/writing-php-code/phpdocs-basics#inline-%40var) that break our tests when things are broken, rather than annotations which don't. Assertions are treated like comments in productions.

```suggestion
    $query = $this->queryHelper->getQuery();
    assert($query instanceof QueryBase, "The query helper getQuery method didn't return an entity query. This is an implementation error");
```